tests/tests.go

```diff
@@ -188,6 +175,38 @@ func setRange(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func setRangeInRangeConfigMap(rangeStart, rangeEnd string) error {
+	configMap, err := testClient.KubeClient.CoreV1().ConfigMaps(ManagerNamespce).Get("kubemacpool-mac-range-config", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	configMap.Data["RANGE_START"] = rangeStart
+	configMap.Data["RANGE_END"] = rangeEnd
+
+	_, err = testClient.KubeClient.CoreV1().ConfigMaps(ManagerNamespce).Update(configMap)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func initKubemacpoolParams(rangeStart, rangeEnd string, namespaceLabelMap map[string]string) error {
+	err := setRangeInRangeConfigMap(rangeStart, rangeEnd)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
do you mean `By("Should succeed setting range in the range config map")`?

Reply:
No. You can add an explanation to `Expect`. In case it fails, it would be clear which expectation has not been fulfilled.

`Expect(err).ToNot(HaveOccurred(), "Should succeed setting range in the KubeMacPool config map")`

Reply:
got it. thx